### ISSUE TITLE
Question on the way implicit objectness is combined with class logits

### ISSUE THREAD

**Issue by thisisi3** [chensnathan/YOLOF#33]
https://github.com/chensnathan/YOLOF/blob/a44b7af0efa9b89d3e91db70bc9a39e49d5d0144/yolof/modeling/decoder.py#L105

Could you please explain why implicit objectness is combined with class logits like this? Thanks.

**Comment by kouwasyou** [comment]
I have the same question, according to the paper, it should be multiply.
